Libraries/SDK/Sources/TIMer/TIMdefs.h

```diff
@@ -123,7 +123,7 @@ extern "C"
  */
 _inline_ ULONG _fastcall_ TIM_ul_GetLowPartTimerInternalCounter(void)
 {
-#if defined( _MSC_VER ) && defined( _WIN32 )
+#if defined( _MSC_VER ) && !defined( _WIN64 )
```

Comment:
IIRC `RDTSC` may provide wonky values on multi-core systems due to the counter not being synchronised between cores, might be worth dropping this branch and always using `GetTickCount()`.